recipe/recipe.yaml

```diff
@@ -13,11 +13,16 @@ source:
       - patches/update-cores.template_NET.csproj.patch
 
 build:
-  number: 0
+  number: 1
+  skip: ppc64le
 
 requirements:
   build:
-    - dotnet
+    - if: not unix
+      then:
+        - dotnet >=8.0.302
+      else:
+        - dotnet >=8.0.302
```

Comment:
We can just use `dotnet >=8.0.302` for all platforms now.

Reply:
Oh right, that's a remnants of trying to figure out why win build was failing